install

```diff
@@ -94,8 +94,12 @@ end
 def sudo?
   return @have_sudo unless @have_sudo.nil?
 
-  Kernel.system "/usr/bin/sudo", "-v"
-  @have_sudo = $CHILD_STATUS && $CHILD_STATUS.success?
+  begin
+    Kernel.system "/usr/bin/sudo", "-l", "mkdir"
+    @have_sudo = $CHILD_STATUS && $CHILD_STATUS.success? || false
+  rescue Interrupt
+    exit!
```

Comment:
```suggestion
    exit
```
`exit` also prevents the traceback.

Reply:
As `SystemExit` trap does not exist, they work almost the same.
But maybe `exit` is better for the future update.

The difference here is just a default exit code (1 or 0).
I thought it should be an error exit, but you can decide later if necessary.

I changed it to `exit`.

Reply:
Good point. `exit 1` is probably better.
```suggestion
    exit 1
```